.travis.yml

```diff
@@ -16,6 +16,7 @@ python:
     - 3.4
     - 3.5
     - 3.6
+    - 3.7
```

Comment:
I'm not sure travis supports this yet, try to use PYTHON_VERSION=3.7 instead lower down in the environment to tell conda to use that version. Even if it would we would install out conda version, too...